templates/signup.html

```diff
@@ -104,6 +104,45 @@ <h4 class="text-blue-500 mt-0 mb-0 text-center">{{_('create_student_account_expl
                         {% endfor %}
                     </select>
                 </div>
+                <!-- START OF NEW QUESTION -->
+                {% if is_teacher %}
+                <div class="signup-container">
+                    <label class="signup-label" for="heard_about">{{_('heard_about_hedy')}}</label>
+                    <ul class="signup-input appearance-none">
+                        <li class="flex" style="justify-content: end;">
```

Comment:
Please style these using Tailwind instead of hard-coded CSS. `class="flex justify-end"` should do the trick!

Reply:
Applied changes! I had these in-line styles only for my current design; however, I don't need them if we are aligning the content to the left instead of the right, so I removed all in-line styles.